averbis/core/_rest_client.py

```diff
@@ -969,6 +959,33 @@ def export_text_analysis_to_cas(
             typesystem=cas_type_system,
         )
 
+    def _load_typesystem(
+            self,
+            type_system: TypeSystem,
+            document_collection: "DocumentCollection",
+            document_name: str
+    ) -> Tuple[TypeSystem, Optional[str]]:
+        document_identifier = None
+        cas_type_system = type_system
+        if cas_type_system is None:
+            try:
```

Comment:
With the additional buildInfo like this
```
{'specVersion': '6.18.0-SNAPSHOT',
 'buildNumber': '25bf16dde4e904dae72d7bebed30f6b9251844f0',
 'platformVersion': '6.53.0'}
 ```

 can we now replace the `try` ... `except` with an if-statement based on the platform version?